projects/bp-gallery/src/components/views/geoguessr/GeoView.tsx

```diff
@@ -63,18 +68,29 @@ const GeoView = () => {
   const archives: FlatArchiveTag[] | undefined =
     useSimplifiedQueryResponseData(picturesData)?.archiveTags;
 
+  const isGeoCollectionPictures = useFlag('geopictures_from_collection');
+  const geoCollectionId = '357';
+  const { data: geoCollectionPictureData } = useGetPicturesForCollectionQuery({
+    variables: { collectionId: geoCollectionId },
+  });
+  const geoCollectionPictureIds: string[] | undefined = useSimplifiedQueryResponseData(
+    geoCollectionPictureData
+  )?.collection?.pictures?.map(picture => picture?.id ?? '');
```

Comment:
The linter doesn't like this since you call `map` on `any`. You can refactor this like so:
```ts
const geoCollection: FlatCollection | undefined =
  useSimplifiedQueryResponseData(geoCollectionPictureData)?.collection;
const geoCollectionPictureIds: string[] | undefined = geoCollection?.pictures?.map(
  picture => picture?.id ?? ''
);
```